### ISSUE TITLE
load extension at startup

### ISSUE THREAD

**Issue by djangoliv** [JoaoFelipe/ipython-unittest#4]
I made several attempts but nothing works. I have to use  %load_ext ipython_unittest.

I try to modify /home/ogiorgis/.jupyter/jupyter_notebook_config.py with

```
c.InteractiveShellApp.extensions = [ 'ipython-unittest']
c.NotebookApp.nbserver_extensions = {'ipython_unittest': True}
```

I also try:
jupyter nbextension enable --py --sys-prefix ipython-unittest 

Is there a solution for not having to do the %load_ext before using %%unittest_main ?

Regards

**Comment by JoaoFelipe** [comment]
I have never tried it. Do you load another extension at startup? Maybe I can look into it to check what is preventing this extension to be loaded.

By the way, the first line in your `jupyter_notebook_config.py` snippet probably should use an underscore:
```
c.InteractiveShellApp.extensions = [ 'ipython_unittest']
```

**Comment by djangoliv** [comment]
No, I do not load another ipython extension (and the jupyterlab extensions have different behavior).

Here is the doc that I tried to follow:

https://ipython.readthedocs.io/en/stable/config/extensions/

Good catch for your remark, but it does not work better with an underscore. 

thanks
